### PR TITLE
Adding LERP option to RotateToAction #4154

### PR DESCRIPTION
**Pull request by wiverson**
First PR, lmk if there is anything I should change.  :)

### CODE REVIEW

**Comment by mgsx-dev** [comment]
Interpolation (not only linear) is already supported. Use [setInterpolation](https://github.com/wiverson/libgdx/blob/c7bb9eeaa7acfa7bf4528275e3da6a73107d8320/gdx/src/com/badlogic/gdx/scenes/scene2d/actions/TemporalAction.java#L122) method on any TemporalAction like RotateTo and RotateBy.

**Comment by MobiDevelop** [comment]
Lerp isn't the right name for what this does, this would be something like useShorterDirection.

**Comment by cypherdare** [comment]
How about `shorterPath`?

File: gdx/src/com/badlogic/gdx/scenes/scene2d/actions/RotateToAction.java [hunk 1]
```diff
@@ -16,17 +16,41 @@
...
+		this.setLerp(useLerp);
+	}
+
 	protected void begin () {
 		start = target.getRotation();
 	}
 
 	protected void update (float percent) {
-		target.setRotation(start + (end - start) * percent);
+		if (isLerp())
```
Comment (cypherdare) [inline gdx/src/com/badlogic/gdx/scenes/scene2d/actions/RotateToAction.java]: Don't need a method call here. Though I suppose JIT will optimize it anyway.

**Comment by wiverson** [comment]
useShortAngle?

**Comment by wiverson** [comment]
FWIW, useLerp refers to the implementation, all of the others are for the user (which is better).  I don't have a strong option on the specific term, I'll submit a better user-centric name tomorrow with whatever seems like a good pick.  :)

**Comment by badlogic** [comment]
Thanks!

**Pull request merged**